packages/replay/src/types.ts

```diff
@@ -108,11 +108,11 @@ export interface SourceMapEntry {
   originalSources: OriginalSourceEntry[];
 }
 
-export interface RecordingEntry {
+export interface RecordingEntry<TMetadata extends UnstructuredMetadata = UnstructuredMetadata> {
```

Comment:
I added a default to all of the touched types to keep this change backwards compatible